Hymns/DetailHymnLyrics.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  DetailHymnLyrics.swift
+//  Hymns
+//
+//  Created by Benjamin Findeisen on 3/28/20.
+//  Copyright © 2020 skywalkerdude. All rights reserved.
+//
+
+import SwiftUI
+
+struct DetailHymnLyrics: View {
+    var body: some View {
+        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
```

Comment:
What is @START_MENU_TOKEN@ and @END_MENU_TOKEN@?

Reply:
Wierd... That isn't even in the file on my local...